src/core/cgroup.c

```diff
@@ -5106,7 +5106,7 @@ static int unit_cgroup_freezer_kernel_state(Unit *u, FreezerState *ret) {
 
 int unit_cgroup_freezer_action(Unit *u, FreezerAction action) {
         _cleanup_free_ char *path = NULL;
-        FreezerState target, current, next;
+        FreezerState current, next, objective;
```

Comment:
shouldn't this change be in the "core/unit: rename freezer "target" to "objective"" commit rather than the "core/cgroup: skip freezer action wholly if current == objective" commit?